crates/oxc_linter/src/snapshots/valid_expect.snap

```diff
@@ -0,0 +1,531 @@
+ 4 │                     return expect(Promise.resolve(1)).toReject();
+   ·                            ─────────────────────────────────────
+ 5 │                 });
+   ╰────
+  help: "Add `await` to your assertion."
+
+  ⚠ eslint(jest/valid-expect): "Promises which return async assertions must be awaited."
+   ╭─[valid_expect.tsx:2:1]
+ 2 │                 test('valid-expect', () => {
+ 3 │                     Promise.resolve(expect(Promise.resolve(2)).resolves.not.toBeDefined());
+   ·                     ──────────────────────────────────────────────────────────────────────
+ 4 │                 });
+   ╰────
+  help: "Add `await` to your assertion."
+
+  ⚠ eslint(jest/valid-expect): "Promises which return async assertions must be awaited."
+   ╭─[valid_expect.tsx:2:1]
+ 2 │                 test('valid-expect', () => {
+ 3 │                     Promise.reject(expect(Promise.resolve(2)).resolves.not.toBeDefined());
+   ·                     ─────────────────────────────────────────────────────────────────────
+ 4 │                 });
+   ╰────
+  help: "Add `await` to your assertion."
+
+  ⚠ eslint(jest/valid-expect): "Promises which return async assertions must be awaited."
+   ╭─[valid_expect.tsx:2:1]
+ 2 │                 test('valid-expect', () => {
+ 3 │                     Promise.x(expect(Promise.resolve(2)).resolves.not.toBeDefined());
+   ·                     ────────────────────────────────────────────────────────────────
+ 4 │                 });
+   ╰────
+  help: "Add `await` to your assertion."
+
+  ⚠ eslint(jest/valid-expect): "Promises which return async assertions must be awaited."
+   ╭─[valid_expect.tsx:2:1]
+ 2 │                 test('valid-expect', () => {
+ 3 │                     Promise.resolve(expect(Promise.resolve(2)).resolves.not.toBeDefined());
+   ·                     ──────────────────────────────────────────────────────────────────────
+ 4 │                 });
+   ╰────
+  help: "Add `await` to your assertion."
+
+  ⚠ eslint(jest/valid-expect): "Promises which return async assertions must be awaited."
+   ╭─[valid_expect.tsx:2:1]
+ 2 │                     test('valid-expect', () => {
+ 3 │ ╭─▶                 Promise.all([
+ 4 │ │                       expect(Promise.resolve(2)).resolves.not.toBeDefined(),
+ 5 │ │                       expect(Promise.resolve(3)).resolves.not.toBeDefined(),
+ 6 │ ╰─▶                 ]);
+ 7 │                     });
+   ╰────
+  help: "Add `await` to your assertion."
+
+  ⚠ eslint(jest/valid-expect): "Promises which return async assertions must be awaited."
+   ╭─[valid_expect.tsx:2:1]
+ 2 │                     test('valid-expect', () => {
+ 3 │ ╭─▶                 Promise.all([
+ 4 │ │                       expect(Promise.resolve(2)).resolves.not.toBeDefined(),
+ 5 │ │                       expect(Promise.resolve(3)).resolves.not.toBeDefined(),
+ 6 │ ╰─▶                 ]);
+ 7 │                     });
+   ╰────
+  help: "Add `await` to your assertion."
```

Comment:
There is duplicated report in this satiation. They all come from the argument of `Promise.all`, is there some good way to only report the first item?